operator/handler/annotations.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handler
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	defaultAnnotationKeyPrefix = "tailing-sidecar.sumologic.com"
```

Comment:
Can this just be sumologic.com so that it matches up with the default [here](https://github.com/SumoLogic/sumologic-kubernetes-fluentd/blob/main/fluent-plugin-kubernetes-sumologic/lib/fluent/plugin/filter_kubernetes_sumologic.rb#L35)?

Reply:
I prefer to change default configuration in [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/conf/logs/fluentd/logs.kubernetes.metadata.filter.conf#L1).